### PR TITLE
Check sample file loaded

### PR DESCRIPTION
**Pull request by LukeLoze1**
Motivation
Desire to setup action rules to run instantly after all cases are loaded from file into case them action db.

Altered
Changed the test around to create the action rule after the "case.sample.inbound" &  "case.action" queues are emptied

In sample files there is a new 100_per_treatment_code,  this takes about 2 mins 15 to run locally and is good for testing changes.

How to test
Get master docker dev running locally, then get this branch and 'make test' . The tests should pass successfully

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -38,6 +40,15 @@ class Config:
...
+                                      PROJECT_PATH.joinpath('resources', 'sample_files',
+                                                            '100_per_treatment_code.csv')))
+
+    # SAMPLE_FILE_PATH = Path(os.getenv('SAMPLE_FILE_PATH',
+    #                                   PROJECT_PATH.joinpath('resources', 'sample_files',
+    #                                                         '1000_per_treatment_code.csv')))
+
+    # SAMPLE_FILE_PATH = Path(os.getenv('SAMPLE_FILE_PATH',
+    #                                   PROJECT_PATH.joinpath('resources', 'sample_files',
+    #                                                         '350000_sample_file.csv')))
```
Comment (AdamHawtin) [inline config.py]: Commented out code

Comment (LukeLoze1) [reply config.py]: I've left those on purpose at the moment to easily switch between files whilst the tests are developed.   Not sure what we're want the default one to be.

Comment (AdamHawtin) [reply config.py]: Should we not be doing that with the environment variable, rather than with commented out code?

Comment (LukeLoze1) [reply config.py]: don't know really, got rid of them now

File: features/steps/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class RabbitConnectionClosedError(Exception):
```
Comment (AdamHawtin) [inline features/steps/exceptions.py]: I think this is duplicated/unused now?

**Pull request merged**